### PR TITLE
Chrometober 2022

### PR DESCRIPTION
**Pull request by jh3y**
- Opening draft PR of Chrometober to see how web.dev handles it 👍

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -297,6 +297,12 @@ module.exports = function (config) {
     }
   }
 
+  // Chrometober config
+  config.addPassthroughCopy({
+    'src/site/content/en/chrometober-2022/': 'chrometober-2022',
```
Comment (devnook) [inline .eleventy.js:302]: You don't want this to be copied over (it is an .njk file)

File: src/site/_includes/partials/script-loader-embed.njk [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{# Sourced scripts cannot be authorized by a CSP hash individually, so we need
+to use an inline script loader for any sourced scripts. #}
+{# This works because the CSP includes the 'strict-dynamic' keyword, allowing
+the children of an authorized script to run. #}
+{% set scriptLoader %}
+function loadScript(url, type) {
```
Comment (devnook) [inline src/site/_includes/partials/script-loader-embed.njk:6]: Is this different from https://github.com/GoogleChrome/web.dev/blob/85422a4f1c431f4ca2ce60a7e8f9411f5d5e0e5a/src/site/_includes/partials/script-loader.njk#L12 ?

File: .eslintignore [hunk 3]
```diff
@@ -3,3 +3,5 @@
 dist/*
 types/*
 /src/site/content/en/patterns/**
+/src/site/content/en/chrometober-2022/**
```
Comment (devnook) [inline .eslintignore:6]: This should not be necessary.

File: src/site/content/en/third_party/chrometober-2022/hoisted.690feb9c.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
...nction(i){for(var n=1;n<arguments.length;n++){var e=arguments[n];for(var t in e)Object.prototype.hasOwnProperty.call(e,t)&&(i[t]=e[t])}return i},b.apply(this,arguments)}function T(i,n){i.prototype=Object.create(n.prototype),i.prototype.constructor=i,w(i,n)}function w(i,n){return w=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e},w(i,n)}function k(i){if(i===void 0)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return i}function I(i,n){(n==null||n>i.length)&&(n=i.length);for(var e=0,t=new Array(n);e<n;e++)t[e]=i[e];return t}function W(i,n){var e=typeof Symbol<"u"&&i[Symbol.iterator]||i["@@iterator"];if(e)return(e=e.call(i)).next.bind(e);if(Array.isArray(i)||(e=function(r,o){if(r){if(typeof r=="string")return I(r,o);var a=Object.prototype.toString.call(r).slice(8,-1);return a==="Object"&&r.constructor&&(a=r.constructor.name),a==="Map"||a==="Set"?Array.from(r):a==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(a)?I(r,o):void 0}}(i))||n&&i&&typeof i.length=="number"){e&&(i=e);var t=0;return function(){return t>=i.length?{done:!0}:{done:!1,value:i[t++]}}}throw new TypeError(`Invalid attempt to iterate non-iterable instance.
```
Comment (devnook) [inline src/site/content/en/third_party/chrometober-2022/hoisted.690feb9c.js:1]: I think we need a license of this file. We can probably put a license for whole dir in third_party/chrometober-2022/LICENSE

**Review by devnook — CHANGES_REQUESTED** [review]
Reviewed

File: src/site/content/en/third_party/chrometober-2022/hoisted.690feb9c.js (hunk 4 above)
Comment (jh3y) [reply src/site/content/en/third_party/chrometober-2022/hoisted.690feb9c.js]: Added 👍

File: .eslintignore (hunk 3 above)
Comment (jh3y) [reply .eslintignore]: Removed 👍

File: .eleventy.js (hunk 1 above)
Comment (jh3y) [reply .eleventy.js:302]: It generates the HTML file in the given directory, no? As in `src/site/content/en/chrometober-2022/` creates an `index.html` at `chrometober-2022` which is what we want?

File: src/site/_includes/partials/script-loader-embed.njk (hunk 2 above)
Comment (jh3y) [reply src/site/_includes/partials/script-loader-embed.njk]: I've rolled our own stripped down version from the Chrometober repo instead.

**Review by devnook — APPROVED** [review]

**Pull request merged**